packages/calypso-e2e/src/lib/pages/writing-settings-page.ts

```diff
@@ -0,0 +1,116 @@
+import { Page, Locator } from 'playwright';
+
+type Sections =
+	| 'Composing'
+	| 'Media'
+	| 'Content types'
+	| 'Feed settings'
+	| 'Theme enhancements'
+	| 'Widgets'
+	| 'Publishing Tools'
+	| 'WordPress.com toolbar';
+type ToggleLevel = 'parent' | 'child';
+
+const selectors = {
+	// Section
+	sectionHeader: ( name: Sections ) => `.section-header :text("${ name }")`,
+
+	// Toggles
+	parentToggle: ( text: string ) => `span:has(:text("${ text }")) .components-form-toggle`,
+	childToggle: ( text: string ) =>
+		`div.site-settings__child-settings:has(label:text("${ text }")) .components-form-toggle`,
```

Comment:
I struggled with and was ultimately unable to make the one selector for both levels of the toggles.